content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -188,11 +188,12 @@ The following tables list all supported operating systems with the name of platf
 | :------- | :----------------------- |
 | Windows  | `windows`                |
 
-##### Linux (Alpine)
+##### Linux (Alpine) x86_64
```

Comment:
Just want to understand why - x86_64 was added - is this the only architecture that Alpine supports, or the only one that we support, or something?

Reply:
It's the only architecture we support for Alpine for now

Reply:
Suggesting something like this to spell it out. Not sure what the best option is but it definitely looks odd after the bracket:

```suggestion
##### Linux (Alpine)

Currently supported for x86_64 only.
```

Reply:
Looks good 👍🏼

Reply:
(just noticed that it explains exactly what I asked in the issue description)